modules/iotacss-utils-display/_utilities.display.scss

```diff
@@ -8,7 +8,7 @@ $iota-utils-display                       : true;
 $iota-utils-display-namespace             : '' !default;
 $iota-utils-display-flex-name             : 'flex' !default;
 $iota-utils-display-block-name            : 'block' !default;
-$iota-utils-display-hidden-name           : 'hidden' !default;
+$iota-utils-display-none-name             : 'none' !default;
```

Comment:
This change will cause the generated class to look like '.u-none'. I think is not descriptive enough.

I would change the:

```
$iota-utils-display-namespace: 'display';
```

but this will make all classes too long, like '.u-display-inline-block'. Let's hold on this for now till there is a better solution.